plugins/GSdx/GSSetting.cpp

```diff
@@ -122,6 +122,13 @@ const char* dialog_message(int ID, bool* updateText) {
 		case IDC_TC_DEPTH:
 			return "Disable the support of Depth buffer in the texture cache.\n"
 				"It can help to increase speed but it will likely create various glitches.";
+		case IDC_8BIT_CPU_FBHACK:
+			return "Forces 8-bit shader conversion to be done on the CPU instead of the GPU but performance will be slower.\n"
+				"Direct3D doesn't support shader conversion, OpenGL does support shader conversion but it doesn't render some corner cases properly.\n"
```

Comment:
Is the text correct? `8-bit shader conversion` - I'm guessing it's supposed to be texture conversion? I'm not sure the shader situation needs to be explained in the tooltip. So something like:

> Uses the CPU instead of the GPU to convert 8-bit textures. May significantly reduce performance.
Fixes glitches or broken graphics in games such as blahdeblahdeblah

Reply:
Yes it should "8 bits frame buffer"